backend.native/compiler/ir/backend.native/src/org/jetbrains/kotlin/backend/konan/lower/FunctionInlining.kt

```diff
@@ -189,7 +188,7 @@ internal class FunctionInlining(val context: Context): IrElementTransformerWithC
 
                 else ->
                     context.ir.originalModuleIndex.functions[descriptor]?.let { it to false }
-                            ?: deserializer.deserializeInlineBody(descriptor)?.let { it as IrFunction to true }
+                            ?: context.ir.symbols.symbolTable.referenceFunction(descriptor).owner to true
```

Comment:
I think there should be at least TODO about getting rid of this function

Reply:
done